ecosystem/typescript/sdk/src/aptos_types/type_tag.ts

```diff
@@ -190,17 +190,16 @@ export class StructTag {
    * @returns
    */
   static fromString(structTag: string): StructTag {
-    // Type args are not supported in string literal
-    if (structTag.includes("<")) {
-      throw new Error("Not implemented");
-    }
-
-    const parts = structTag.split("::");
-    if (parts.length !== 3) {
-      throw new Error("Invalid struct tag string literal.");
-    }
-
-    return new StructTag(AccountAddress.fromHex(parts[0]), new Identifier(parts[1]), new Identifier(parts[2]), []);
```

Comment:
we don't need `new Identifier`?

Reply:
Hmm, I think the `module_name` and `name` from line 199-200 are actually Identifier already